tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -1131,6 +1131,14 @@ public TezConfiguration(boolean loadDefaults) {
 
   public static final int TASK_HEARTBEAT_TIMEOUT_CHECK_MS_DEFAULT = 30 * 1000;
 
+  /**
+   * String value. Comma seperated list of FileSystem paths which needs to be eagerly initialized.
+   * For example s3://bucket/,file://,hdfs://localhost:8020/
+   */
+  @ConfigurationScope(Scope.AM)
```

Comment:
1. Scope is VERTEX as we're doing this in TezChild
2. constant name should contain TEZ_ prefix

Reply:
Ack.

Reply:
ack.